cli/tests/pcluster/config/test_validators.py

```diff
@@ -2018,3 +2019,39 @@ def test_fsx_ignored_parameters_validator(mocker, section_dict, expected_error):
         assert_that(errors[0]).matches(expected_error)
     else:
         assert_that(errors).is_empty()
+
+
+@pytest.mark.parametrize(
+    "section_dict, expected_error",
+    [
+        ({"volume_type": "standard", "volume_size": 15}, None),
+        ({"volume_type": "standard", "volume_size": 0}, "The volume size of standard must be at least 1 GiB"),
+        ({"volume_type": "standard", "volume_size": 1025}, "The volume size of standard must be at most 1 TiB"),
+        ({"volume_type": "io1", "volume_size": 15}, None),
+        ({"volume_type": "io1", "volume_size": 3}, "The volume size of io1 must be at least 4 GiB"),
+        ({"volume_type": "io1", "volume_size": 16385}, "The volume size of io1 must be at most 16 TiB"),
+        # TODO Uncomment these lines agter making io2 supported
```

Comment:
minor: typo, agter should be after

also, instead of 'making io2 supported' consider changing it to 'adding support for io2 volume types'

Reply:
Yeah! It is done